libs/expo/betterangels/src/lib/screens/AddEditClient/HouseholdMembers/Household/index.tsx

```diff
@@ -0,0 +1,150 @@
+import { Colors, Radiuses, Spacings } from '@monorepo/expo/shared/static';
```

Comment:
this is more a comment for @shiozakiyuko: if we're bolding the values ("May", "Other", etc.) , we should probably bold and/or increase the size of the header ("Aunt"). 

<img width="295" alt="image" src="https://github.com/user-attachments/assets/5cc6abc7-75c1-4773-ba54-7e05dc1eebb2">